src/pages/catalog/oer/index.tsx

```diff
@@ -0,0 +1,244 @@
+import React from "react";
+import Head from "next/head";
+import Link from "next/link";
+import Image, { ImageProps } from "next/image";
+
+import { PageBanner } from "../../../components/Banner";
+import { Heading } from "../../../components/Heading";
+import { RequestCourseModal } from "../../../components/RequestCourseModal";
+
+import styles from "./index.module.scss";
+import eorstatics from "../../../lib/images/tam211.png";
+import eordynamics from "../../../lib/images/tam212.png";
+import eorsolids from "../../../lib/images/tam251.png";
+import eornumerical from "../../../lib/images/cs357.png";
+import eorphysics1 from "../../../lib/images/EORphysics1.png";
+import eorphysics2 from "../../../lib/images/EORphysics2.png";
+import eorpython from "../../../lib/images/EORpython.png";
+import eorthermo from "../../../lib/images/EORthermo.png";
+import sigcse2023 from "../../../lib/images/sigcse2023.png";
+import cs233Image from "../../../lib/images/cs-233.png";
+import ece6353fraida from "../../../lib/images/ece6353fraida.png";
+
+interface SharedQuestionCardProps {
+  image: ImageProps["src"];
+  title: string;
+  href: string;
+  institution?: string;
+  ownerName?: string;
+  ownerEmail?: string;
+  github?: string;
+}
+
+const SharedQuestionCard: React.FC<SharedQuestionCardProps> = ({
+  image,
+  title,
+  href,
+  institution,
+  ownerName,
+  ownerEmail,
+  github,
+}) => {
+  return (
+    <article className="card border-secondary overflow-hidden">
+      <Link href={href} className="position-relative">
+        <Image
+          src={image}
+          alt={title}
+          style={{
+            objectFit: "contain",
+            width: "100%",
+            height: "100%",
+            aspectRatio: "5 / 3",
+          }}
+        />
+      </Link>
+      <div className="card-body">
+        <Link href={href}>
+          <h3 className="card-title h5">{title}</h3>
+        </Link>
+        {ownerName && (
+          <p className="mb-1">
+            <strong>Shared by:</strong> {ownerName}{" "}
+            {ownerEmail && <a href={`mailto:${ownerEmail}`}>{ownerEmail}</a>}
+          </p>
+        )}
+        {institution && (
+          <p className="mb-1">
+            <strong>Institution: </strong> {institution}
+          </p>
+        )}
+        {github && (
+          <p className="mb-1">
+            <strong>View source</strong> on <Link href={github}>Github</Link>
```

Comment:
There's no need for the first part to be bold, and making "GitHub" the only text in the link makes this needlessly difficult to use for folks using screen readers.

```suggestion
            <Link href={github}>View source on Github</Link>
```